packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-cosmosdb-sql-database.ts

```diff
@@ -0,0 +1,22 @@
+import { TerraformStack } from 'cdktf'
+import { CosmosdbAccount, CosmosdbSqlDatabase, ResourceGroup } from '@cdktf/provider-azurerm'
+import { toTerraformName } from '../helper/utils'
+import { BoosterConfig } from '@boostercloud/framework-types'
+
+export class TerraformCosmosdbSqlDatabase {
+  static build(
+    terraformStack: TerraformStack,
+    resourceGroup: ResourceGroup,
+    appPrefix: string,
+    cosmosdbAccount: CosmosdbAccount,
+    config: BoosterConfig
+  ): CosmosdbSqlDatabase {
+    const idDatabase = toTerraformName(appPrefix, 'dbd')
+    return new CosmosdbSqlDatabase(terraformStack, idDatabase, {
+      name: config.resourceNames.applicationStack,
+      resourceGroupName: resourceGroup.name,
+      accountName: cosmosdbAccount.name,
+      throughput: 400,
```

Comment:
Is there a way to configure the throughput of both the events store and the read models store to be automatic?
I mean, Is there an option for auto-scalability?